src/middlewared/middlewared/plugins/boot.py

```diff
@@ -49,7 +49,11 @@ async def get_disks_cache(self):
         if BOOT_POOL_DISKS is None:
             # Using an immutable object is very important since this is
             # a globally cached value
-            BOOT_POOL_DISKS = tuple(await self.middleware.call('zfs.pool.get_disks', BOOT_POOL_NAME))
+            disks = list()
+            args = {'name': BOOT_POOL_NAME, 'real_paths': True}
```

Comment:
Do we need to be defensive and only execute this if `BOOT_POOL_NAME` is not `None`?

Reply:
I just thought about that before I merged, but I figured it's best to just crash since the `setup()` functions should be called before the `boot.get_disks` endpoint is called.